mmv1/templates/terraform/examples/compute_rmig_resize_request.tf.tmpl

```diff
@@ -0,0 +1,77 @@
+resource "google_compute_region_instance_template" "a3_dws" {
+  provider = google-beta
+  name                 = "a3-dws"
+  region               = "us-central1"
+  description          = "This template is used to create a mig instance that is compatible with DWS resize requests."
+  instance_description = "A3 GPU"
+  machine_type         = "a3-highgpu-8g"
+  can_ip_forward       = false
+
+  scheduling {
+    automatic_restart   = false
+    on_host_maintenance = "TERMINATE"
+  }
+
+  disk {
+    source_image = "cos-cloud/cos-105-lts"
+    auto_delete  = true
+    boot         = true
+    disk_type    = "pd-ssd"
+    disk_size_gb = "960"
+    mode         = "READ_WRITE"
+  }
+
+  guest_accelerator {
+    type  = "nvidia-h100-80gb"
+    count = 8
+  }
+
+  reservation_affinity {
+    type = "NO_RESERVATION"
+  }
+
+  shielded_instance_config {
+    enable_vtpm                 = true
+    enable_integrity_monitoring = true
+  }
+
+  network_interface {
+    network = "default"
+  }
+}
+
+resource "google_compute_region_instance_group_manager" "a3_dws" {
+  provider = google-beta
+  name               = "a3-dws"
```

Comment:
Same here. Though you could also use a different variable for this name.

```suggestion
  name                 = "{{index $.Vars "resize_request_name"}}"
```

Reply:
not sure completely how it is work. But modified with the suggestion for template too. Check please.